src/cli/src/cluster/releases.rs

```diff
@@ -0,0 +1,33 @@
+use crate::CliError;
+use super::*;
+use fluvio_cluster::HelmClient;
+
+const DEFAULT_CHART_NAME: &str = "fluvio/fluvio-app";
```

Comment:
this is already defined in here: https://github.com/infinyon/fluvio/blob/master/src/cluster/src/install.rs

Reply:
Well, it's not public from the `fluvio-cluster` crate, and I don't necessarily think it should be part of the public API. I don't see a problem with this being left here.

Reply:
it's duplicate chart name in the same crate.